src/quo/components/list_items/token_value/view.cljs

```diff
@@ -6,15 +6,15 @@
     [quo.components.markdown.text :as text]
     [quo.components.utilities.token.view :as token]
     [quo.foundations.colors :as colors]
-    [quo.foundations.common :as common]
```

Comment:
should we delete the function common/token-label too? or is it still used somewhere?

Reply:
I deleted it :) https://github.com/status-im/status-mobile/pull/18136/files#diff-63a9cce758c7362591b282e7f9db5767fed12cb718ddc00a8a01eb44e3c9e53cL6-L10

Reply:
My bad for not noticing! Thanks 😀